vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/SetContextSteps.java

```diff
@@ -56,7 +61,59 @@ public void switchToNativeContext()
      * @param index index of web view
      */
     @When("I switch to web view with index `$index`")
-    public void switchToWebView(int index)
+    public void switchToWebViewByIndex(int index)
+    {
+        performOnWebViews(webViews ->
+        {
+            int webViewIndex = index - 1;
+            int size = webViews.size();
+            if (webViewIndex >= 0 && webViewIndex < size)
```

Comment:
I think makes sense to throw an exception in case of negative values, it might be out of the scope of this PR btw